workflows/metatdenovo.nf

```diff
@@ -288,7 +288,11 @@ workflow METATDENOVO {
             WRITESPADESYAML.out.yaml,
             []
         )
+        if ( params.spades_flavor == 'rnaviral') {
+            ch_assembly = SPADES.out.contigs
+        } else {
         ch_assembly = SPADES.out.transcripts
+        }
```

Comment:
What's this @danilodileo? Does `rnaviral` have any special settings, or are the output channels different?

Reply:
Moreover, there's probably a mistake in the indentation.